Gemfile.lock

```diff
@@ -79,8 +87,14 @@ GEM
     minitest (5.10.1)
     multi_json (1.12.1)
     nio4r (1.2.1)
+    nokogiri (1.7.0.1)
```

Comment:
Nvm, I figured this one out. There's different versions of nokogiri and other gems because @rmkeezer is difficult and wants to develop on Windows lol